setup-gcloud/README.md

```diff
@@ -1,12 +1,6 @@
-# setup-gcloud
+# setup-gcloud-base
```

Comment:
Shouldn't we continue to name this setup-gcloud?

Reply:
Can we even have both the composite action and the "base" action called setup-gcloud?

Reply:
No, but this README is in setup-gcloud so I think it should have the `setup-gcloud` title :)

Reply:
oh yeah, sorry :)